README.md

```diff
@@ -209,6 +209,7 @@ If you are completely new to Git (_Version Control_) and/or GitHub,
 we suggest you check out the following:
 
 + Beginners Guide to Git: https://guides.github.com/activities/hello-world/
++ [First Contributions](https://github.com/roshanjossey/first-contributions): A project to help you get started with contributing to open source projects
```

Comment:
@Roshanjossey you have created a _good_ resource, 👍 
_however_ I am _not_ a "fan" of how it is React-biassed without a _clear_ reason ... 🙄 
![first-contributions-react-focussed](https://user-images.githubusercontent.com/194400/33946743-75442182-e01a-11e7-850c-3c1468c06975.png)

Reply:
@nelsonic, Thank you for your response.
I really don't have a bias towards react. I like react. Web app for first-contributions is built with react.
That doesn't mean React is the first project I recommend to our users. A lot of people ask for suggestions in our slack group. You can see for yourself there that I mostly suggest exercism. I believe exercism is the best place to start contributing.
So the reason why react comes first is just because it was the first project that came in to my mind when I wrote that list.